.github/workflows/clang-tidy.yml

```
@@ -0,0 +1,88 @@
+name: CLANG_TIDY
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+#addons:
+#  homebrew:
+#    packages:
+#    - git
+#  apt:
+#    sources:
+#    - llvm-toolchain-xenial-8
+#    - ubuntu-toolchain-r-test
+#    packages:
+#    - llvm-8
+#    - g++-8
+#    - clang-8
+#    - git
+#    - python3
+#    - python3-pip
+#    - python3-setuptools
+#    - clang-tidy-8
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [linux]
```

```suggestion
        os: [ubuntu-latest]
```
https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#jobsjob_idruns-on